linode/manager
# test: [M3-6221] - Clean up `cy.intercept` calls in support ticket create test

## Description 📝
Quick clean up to replace calls to `cy.intercept` with utils in our support ticket create test.

## Changes  🔄
- Replace calls to `cy.intercept` with mock utils in `open-support-ticket.spec.ts`.
- Adds `mockAttachSupportTicketFile()` util

## How to test 🧪
We can rely on CI to confirm that this test has not been broken by these changes, but to run the test locally you can use this command:

```bash
yarn cy:run -s "cypress/e2e/core/helpAndSupport/open-support-ticket.spec.ts"
```

## As an Author I have considered 🤔

*Check all that apply*
